docs/guides/advanced/link-to-code-source.md

```diff
@@ -0,0 +1,45 @@
+!!! warning "Experimental"
+    This feature is experimental and may change in the future.
+
+    Please give us feedback on [Slack][slack].
+
+We support linking to the source code on GitHub, or any other VCS provider.
+
+![Link to GitHub](../../images/guide/link-to-github.gif)
+
+## Usage
+
+To enable source code linking with Logfire, you need to configure your project. Here's how:
+
+```python
+import logfire
+
+logfire.configure(
+    code_source=logfire.CodeSource(
+        repository='https://github.com/pydantic/logfire',
```

Comment:
This (combined with "We support linking to the source code on GitHub, or any other VCS provider") makes it look like it'll just work if they provide a non-github link, which it likely won't.

Reply:
What do you suggest?

Reply:
It looks like our URL construction strategy works for gitlab (e.g. https://gitlab.com/rluna-gitlab/gitlab-ce/blob/v16.0.10/yarn.lock#L13) so i guess we support that too? maybe we just say we support github, gitlab, and any other VCS provider that uses the same URL format? we clearly don't support ANY provider.

Reply:
Ah, ok. I see.

Reply:
changed